doc/api/globals.md

```diff
@@ -245,6 +245,49 @@ added: v0.3.0
 Use the internal `require()` machinery to look up the location of a module,
 but rather than loading the module, just return the resolved filename.
 
+### require.addTransform(transform)
```

Comment:
The entire `.md` file appears to be lacking argument and return type info, actually...